test/onnx_import.cpp

```diff
@@ -413,3 +413,52 @@ TEST(onnx, model_softmax)
     auto result_vectors = execute(function, inputs, "INTERPRETER");
     EXPECT_TRUE(test::all_close_f(expected_output, result_vectors.front()));
 }
+
+TEST(onnx, model_flatten)
+{
+    auto function = onnx_import::import_onnx_function(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/flatten.onnx"));
+
+    Inputs inputs;
+
+    inputs.emplace_back(
+        test::NDArray<float, 4>(
+            {{{{1, 2, 3, 4, 5}, {6, 7, 8, 9, 10}, {11, 12, 13, 14, 15}, {16, 17, 18, 19, 20}},
```

Comment:
Please make the test more compact, by using smaller input and output data.

Reply:
In my opinion we need many dimensions to test flatten properly.

Reply:
Yes, but each can have length of 2 or 3